WinNSMenu.m

```diff
@@ -271,11 +191,15 @@ HMENU r_build_menu_for_itemmap(NSMenu *menu, BOOL asPopUp, BOOL fakeItem, NSMapT
 	{
 	  flags = MF_STRING;
 	  s = menu_tag++;
-	  if(fakeItem)
-	    {
-	      item = [[GSFakeNSMenuItem alloc] initWithItem: item];
-	      AUTORELEASE(item);
-	    }
+   	  if (([item action] == NULL || [item target] == nil) && notPullDown && asPopUp)
```

Comment:
@gcasa @fredkiefer Once the GSFakeMenuItems were removed we were able to click on pop ups but they were inoperable because there was no action set on them. This change here populates the action.